src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/does.not.share.with.different.protected.scope/accept/server.rpt

```diff
@@ -38,32 +39,36 @@ read closed
 write nukleus:begin.ext ${http:header(":status", "200")}
                         ${http:header("content-length", "11")}
                         ${http:header("date", date)}
-                        ${http:header("cache-control", "stale-while-revalidate=2147483648")}
+                        ${http:header("surrogate-control", "max-age=1+10, x-protected")}
+                        ${http:header("cache-control", "stale-while-revalidate=10")}
 
 write flush
+
 write nukleus:data.ext ${http:header(":scheme", "http")}
                        ${http:header(":method", "GET")}
                        ${http:header(":path", "/path")}
                        ${http:header(":authority", "localhost:8080")}
-                       ${http:header("x-retry-after", "2")}
-                       ${http:header("x-poll-injected", "x-http-cache-sync, no-cache")}
+                       ${http:header("authorization", "some-authorization-value")}
                        ${http:header("cache-control", "no-cache")}
-                       ${http:header("x-http-cache-sync", "always")}
+                       ${http:header("prefer", "x-on-update, wait=1")}
```

Comment:
The `x-on-update` preference seems to be missing context about what constitutes an update, i.e. relative to what?

```
${http:header("prefer", "wait=1")}
${http:header("if-none-match", "bfc13a64729c4290ef5b2c2730249c88ca92d82d")}
```

Suggest using `if-none-match` to request the normal behavior, when combined with `wait=1` preference, gives the server a time window to provide an updated response, or fall back to `304` if the waiting period expires.